pkg/providers/aws/provider_blobstorage.go

```diff
@@ -493,9 +506,42 @@ func reconcileS3BucketSettings(bucket string, s3svc s3iface.S3API) error {
 	if err != nil {
 		return errorUtil.Wrapf(err, "failed to set encryption settings on bucket %s", bucket)
 	}
+
+	if (bs.Spec.AllowPublicGetObjectAccess) {
+		policy, err := getAllowPublicGetObjectAccess(bucket)
+		if (err != nil) {
+			return errorUtil.Wrapf(err, "failed to get allow public get object access")
+		}
+
+		_, err = s3svc.PutBucketPolicy(&s3.PutBucketPolicyInput{
+			Bucket: aws.String(bucket),
+			Policy: aws.String(string(policy)),
+		})
+		if err != nil {
+			return errorUtil.Wrapf(err, "failed to create bucket policy on %s", bucket)
+		}
+	}
 	return nil
 }
 
+func getAllowPublicGetObjectAccess(bucketName string) ([]byte, error) {
+
+	s3Policy := map[string]interface{}{
+		"Version": "2012-10-17",
+		"Statement": []map[string]interface{}{
+			{
+				"Sid":    "Stmt1618957960008",
+				"Effect": "Allow",
+				"Principal": "*",
```

Comment:
Do we really need to add get access for everyone, can we not limit this to the cluster? 

Maybe we can add a condition to this to limit it to the cluster VPC at the very least https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-block-public-access.html#access-control-block-public-access-policy-status

Reply:
The S3 URL is hard coded into the HTML of the 3Scale Portal, hence the need to make public